drivers/include/hdc1000.h

```diff
@@ -136,7 +139,32 @@ int hdc1000_trigger_conversion(const hdc1000_t *dev);
  * @return                  HDC1000_OK on success
  * @return                  HDC1000_BUSERR on I2C communication failures
  */
-int hdc1000_get_results(const hdc1000_t *dev, int16_t *temp, int16_t *hum);
+int hdc1000_get_results(hdc1000_t *dev, int16_t *temp, int16_t *hum);
+
+/**
+ * @brief   Check if cached values are outdated or not
+ *
+ * @param[in]  dev          device descriptor of sensor
+ *
+ * @return                  true if cached values are outdated
+ * @return                  false otherwise
+ */
+bool hdc1000_outdated(hdc1000_t *dev);
+
+/**
+ * @brief   Convenience function for reading temperature and humidity
+ *
+ * This function will return cached values if they are valid, or will trigger a new conversion,
+ * wait for the conversion to be finished and the get the results from the device.
```

Comment:
Pleas make clear that this function either
- returns cached data if readings are withing the sampling period introduced above
- **or** a new sensor read is triggered